js/models/smil_model.js

```diff
@@ -1149,13 +1149,13 @@ SmilModel.fromSmilDTO = function(smilDTO, mo) {
                 console.error("SMIL clipEnd is string, parsing float... (" + node.clipEnd + ")");
                 node.clipEnd = parseFloat(node.clipEnd);
             }
-            if (node.clipEnd <= node.clipBegin)
+            if (node.clipEnd < node.clipBegin)
             {
                 if (smilModel.mo.DEBUG)
                 {
-                    console.log(getIndent() + "JS MO clipEnd adjusted to MAX");
+                    console.log(getIndent() + "JS MO clipEnd adjusted to clipBegin + 0.1");
                 }
-                node.clipEnd = node.MAX;
+                node.clipEnd = node.clipBegin + 0.1;
             }
```

Comment:
1) due to this code change, if `clipBegin` == `clipEnd` then the audio player will probably misbehave, right?
2) what is the rationale for this change? Before the change, setting `clipEnd` to MAX (when `clipEnd` <= `clipBegin` ... granted, that's a weird edge-case) informs the audio player keep playback until the natural end of the audio file. With the change, the audio player will play 0.1 seconds of content?